Automattic/wp-calypso
# Stats: Add new skip button to purchase flow

Related to #86131.

## Proposed Changes

Adds a new button to the PWYW flow to allow skipping the purchase flow. This button doesn't do anything yet. That will come in #86136. Just adding the button with appropriate styling/layout here.

## Testing Instructions



* Launch the Live build.
* Visit the Stats page and click the upgrade notice.
* Confirm only the single button labeled, "Get Stats" is visible.
* Enable the feature flag: `stats/checkout-flows-v2`
* Confirm there are now two buttons visible matching the following screenshot.
* Confirm the buttons flow as needed at smaller screen sizes.

<img width="355" alt="SCR-20240117-sgbn" src="https://github.com/Automattic/wp-calypso/assets/40267301/e017eb38-3d8e-4e3a-bf18-1f1df580e1ce">

## Pre-merge Checklist
